controller/heartbeat/heartbeat.go

```diff
@@ -22,11 +22,11 @@ import (
 func K8sValues(kubeAPI *k8s.KubernetesAPI, controlPlaneNamespace string) url.Values {
 	v := url.Values{}
 
-	cm, configPB, err := healthcheck.FetchLinkerdConfigMap(kubeAPI, controlPlaneNamespace)
+	cm, _, err := healthcheck.FetchLinkerdConfigMap(kubeAPI, controlPlaneNamespace)
 	if err != nil {
 		log.Errorf("Failed to fetch linkerd-config: %s", err)
 	} else {
-		v.Set("uuid", configPB.GetInstall().GetUuid())
+		v.Set("uuid", string(cm.GetUID()))
```

Comment:
Interesting point. Would `existing ones` be defined as ones that we have _already seen a heartbeat from_ ?

Reply:
As soon as people start upgrading, the heartbeat, the dashboard and the Grafana will start sending the new uuid. I chatted a bit about this with @grampelberg. We think this will produce some funky stats during one month aprox, while people get on board. @grampelberg can you confirm we can go ahead with this?

Reply:
Yup, go ahead!